cmake/cvars.cmake

```
@@ -0,0 +1,50 @@
+set(CVAR_VSYNC_ENABLED "gVsyncEnabled" CACHE STRING "")
+set(CVAR_Z_FIGHTING_MODE "gZFightingMode" CACHE STRING "")
+set(CVAR_NEW_FILE_DROPPED "gNewFileDropped" CACHE STRING "")
+set(CVAR_DROPPED_FILE "gDroppedFile" CACHE STRING "")
+set(CVAR_INTERNAL_RESOLUTION "gInternalResolution" CACHE STRING "")
+set(CVAR_MSAA_VALUE "gMSAAValue" CACHE STRING "")
+set(CVAR_SDL_WINDOWED_FULLSCREEN "gSdlWindowedFullscreen" CACHE STRING "")
+set(CVAR_TEXTURE_FILTER "gTextureFilter" CACHE STRING "")
+set(CVAR_CONTROL_NAV "gControlNav" CACHE STRING "")
+set(CVAR_CONSOLE_ENABLED "gConsoleEnabled" CACHE STRING "")
+set(CVAR_CONTROLLER_CONFIGURATION_ENABLED "gControllerConfigurationEnabled" CACHE STRING "")
+set(CVAR_CONTROLLER_DISCONNECTED_ENABLED "gControllerDisconnectedWindowEnabled" CACHE STRING "")
+set(CVAR_CONTROLLER_REORDERING_ENABLED "gControllerReorderingWindowEnabled" CACHE STRING "")
+set(CVAR_GFX_DEBUGGER_ENABLED "gGfxDebuggerEnabled" CACHE STRING "")
+set(CVAR_STATS_ENABLED "gStatsEnabled" CACHE STRING "")
```

```suggestion
set(CVAR_CONSOLE_WINDOW_OPEN "gConsoleEnabled" CACHE STRING "")
set(CVAR_CONTROLLER_CONFIGURATION_WINDOW_OPEN "gControllerConfigurationEnabled" CACHE STRING "")
set(CVAR_CONTROLLER_DISCONNECTED_WINDOW_OPEN "gControllerDisconnectedWindowEnabled" CACHE STRING "")
set(CVAR_CONTROLLER_REORDERING_WINDOW_OPEN "gControllerReorderingWindowEnabled" CACHE STRING "")
set(CVAR_GFX_DEBUGGER_WINDOW_OPEN "gGfxDebuggerEnabled" CACHE STRING "")
set(CVAR_STATS_WINDOW_OPEN "gStatsEnabled" CACHE STRING "")
```

i'm open to suggestions here but i think this better describes what these cvars actually do